services/motion/builtin/move_request.go

```diff
@@ -513,13 +514,13 @@ func (ms *builtIn) newMoveOnGlobeRequest(
 		fs,
 		geomsRaw,
 		valExtra,
+		requestTypeMoveOnGlobe,
```

Comment:
Not a problem for this PR I think but we've got a leaky abstraction somewhere if we keep having to pass this around.